crossbeam-channel/examples/stopwatch.rs

```diff
@@ -52,3 +53,8 @@ fn main() {
         }
     }
 }
+
+#[cfg(windows)]
```

Comment:
How about moving this to the top?  In the spirit of early return.. :)

Reply:
Sounds good to me :)